tests/unit_node/process_test.ts

```diff
@@ -243,7 +243,7 @@ Deno.test({
           setInterval(() => {}, 1000);
           const listener = () => {
             console.log("foo");
-            process.off("SIGINT")
+            process.off("SIGINT", listener)
```

Comment:
Err, this seems off. Can you revert it?

Reply:
This is actually required in node.js -- the code won't run w/o it.

Reply:
```
TypeError [ERR_INVALID_ARG_TYPE]: The "listener" argument must be of type function. Received undefined
```

Reply:
Huh, so it didn't fail properly the test even though the required arg was not provided?

Reply:
Correct. The test wasn't really testing anything. I think process.off needs a bit more work.